website/docs/reference/resource-configs/hard-deletes.md

```diff
@@ -0,0 +1,113 @@
+---
+title: hard_deletes
+resource_types: [snapshots]
+description: "Use the `hard_deletes` config to control how deleted rows are tracked in your snapshot table."
+datatype: "{<dictionary>}"
+default_value: {ignore}
+id: "hard-deletes"
+sidebar_label: "hard_deletes"
+---
+
+Available from dbt v1.9 or with [Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
+
+
+<File name='snapshots/schema.yml'>
+
+```yaml
+snapshots:
+  - name: <snapshot_name>
+    config:
+      hard_deletes: 'ignore', 'invalidate', or 'new_record'
+```
+</File>
+
+<File name='dbt_project.yml'>
+
+```yml
+snapshots:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +hard_deletes: "ignore", "invalidate", or "new_record"
+```
+
+</File>
+
+<File name='snapshots/<filename>.sql'>
+
+```sql
+{{
+    config(
+        unique_key='id',
+        strategy='timestamp',
+        updated_at='updated_at',
+        hard_deletes='ignore', 'invalidate', 'new_record'
+    )
+}}
+```
+
+</File>
+
+
+## Description
+
+Use the `hard_deletes` configuration to track hard deletes by adding a new record when row become "deleted" in source. 
+Replaces the `invalidate_hard_deletes` config to give you more control on how to handle deleted rows from the source.
+
+import HardDeletes from '/snippets/_hard-deletes.md';
+
+<HardDeletes />
+
+:::warning
+
+If you're updating an existing snapshot to use the `hard_deletes` config, dbt _will not_ handle migrations automatically. We recommend either only using these settings for net-new snapshots, or arranging an update of pre-existing tables before enabling this setting.
+:::
+
+## Default
+
+By default, if you don’t specify `hard_deletes`, it'll automatically default to `ignore`. Deleted rows will not be tracked and their `dbt_valid_to` column remains `NULL`.
+
+The `hard_deletes` config has three options:
+
+| Field | Description |
+| --------- | ----------- |
+| `ignore` (default) | No action for deleted records. |
+| `invalidate` | Behaves the same as the existing `invalidate_hard_deletes=true`, where deleted records are invalidated by setting `dbt_valid_to`. |
+| `new_record` | Tracks deleted records as new rows using the `dbt_is_deleted` meta field when records are deleted.|
+
+## Impact on snapshot records
```

Comment:
Not sure this is the right header for this section

Reply:
changed it to ## Considerations